code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -180,6 +180,9 @@
 	max_ammo = 8
 	multiple_sprites = 2
 
+/obj/item/ammo_box/magazine/m10mm/empty		//for maint drops
```

Comment:
It does, it then swaps the regular magazine with this one, making the gun only have 1 bullet, but still accept regular 10mm mags if they get made from the protolathe.